docs/source/markdown/podman-stats.1.md.in

```diff
@@ -51,7 +51,7 @@ Valid placeholders for the Go template are listed below:
 | .MemUsageBytes      | Memory usage (IEC)                               |
 | .Name               | Container Name                                   |
 | .NetIO              | Network IO                                       |
-| .Network            | Network I/O, separated by network interface      |
+| .Network ...       | Network I/O, separated by network interface      |
```

Comment:
if you merge https://github.com/containers/podman/pull/21682 and rebase then you do not need to change this here